Library/Homebrew/cmd/tap.rb

```diff
@@ -2,16 +2,20 @@ module Homebrew
   def tap
     if ARGV.empty?
       each_tap do |user, repo|
-        puts "#{user.basename}/#{repo.basename.sub("homebrew-", "")}" if (repo/".git").directory?
+        if (repo/".git").directory?
+          puts "#{user.basename}/#{repo.basename.sub("homebrew-", "")}"
+        end
```

Comment:
Is there a functional change here? If so, what is it? Otherwise I'd prefer to avoid the churn.